modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/ProjectResource.java

```diff
@@ -274,6 +304,12 @@ private TaskResultJson doApplyReadAccess( final ProjectName projectName, final P
             execute();
     }
 
+    private AuthenticationInfo adminAuthInfo()
```

Comment:
Why is this needed?
 I thought if user doesn't have rights to read contents it should not see that repository has such contentId.